ivy/functional/ivy/experimental/activations.py

```diff
@@ -283,6 +320,18 @@ def logsigmoid(
     return ivy.current_backend(input).logsigmoid(input, out=out)
 
 
+def _selu_jax_like(
+    x: Union[ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    fn_original=None,
+    out: Optional[ivy.Array] = None,
+) -> ivy.Array:
+    alpha = 1.6732632423543772848170429916717
+    scale = 1.0507009873554804934193349852946
+    return scale * elu(x, alpha=alpha)
```

Comment:
This line causes errors in `tf`, `torch`, and `paddle`. It might be better to call ivy functions explicitly
```suggestion
    return ivy.multiply(scale, ivy.elu(x, alpha=alpha))
```

Reply:
Done